test/parallel/test-child-process-fork-detached.js

```diff
@@ -0,0 +1,44 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
AFAIK this copyright is not needed for new tests.

Reply:
Will remove it. Thanks for letting me know.